dash/dash.py

```diff
@@ -17,7 +17,6 @@
 from textwrap import dedent
 
 import flask
-from flask import Flask, Response
```

Comment:
ha yeah, we already had a few instances of `flask.Response` in the code below... definitely cleaner this way. Thanks LGTM 👍